src/permissions_template.h

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright (C) 2015 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#ifndef PERMISSIONS_TEMPLATE_H
+#define PERMISSIONS_TEMPLATE_H
+
+#include "process.h"
+
+struct permissions_template {
+    // The owner of the template
+    struct process *owner;
+
+    // File permissions
+    struct list *directories;
+
+    // Memory permissions
+    int max_number_of_pages;
+
+    // Window permissions, to override parents'
+    int max_width;
+    int max_height;
+    int offset_x;
+    int offset_y;
+};
+
+/**
+ * @brief   Creates a new permissions template for a process to use.
+ * @details Allocates an identifier for a process permissions template,
```

Comment:
np: phrasing. We aren't allocating an identifier, but rather a permissions template